extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js

```diff
@@ -0,0 +1,174 @@
+require([
+	'jquery',
+	'mw',
+	'wikia.window',
+	'wikia.loader',
+	'wikia.mustache',
+	'wikia.tracker',
+	'wikia.abTest'
+], function ($, mw, w, loader, mustache, tracker, abTest) {
+	'use strict';
+
+	var $banner,
+		bannerBottomOffset,
+		bannerOffset,
+		dismissCookieName = 'pmp-entry-point-dismissed',
+		experimentGroup = abTest.getGroup('POTENTIAL_MEMBER_PAGE_ENTRY_POINTS'),
+		experiments = {
+			TOP: {
+				type: 'top',
+				addEntryPoint: function () {
+					$banner.insertAfter($('#WikiaPageHeader .header-container'));
+				}
+			},
+			IN_ARTICLE: {
+				type: 'in-article',
+				addEntryPoint: function () {
+					var headers = mw.util.$content.children('h2'),
+						$header;
+
+					// Check if there are headers in content
+					if (headers.length >= 2) {
+						$header = headers.eq(0);
+
+						// Check if first header is not first node in content
+						if ($header.prevAll('p').length) {
+							$banner.insertBefore($header);
+						} else {
+							$banner.insertBefore(headers.eq(1));
+						}
+					} else {
+						mw.util.$content.append($banner);
+					}
+				}
+			},
+			GLUE_BOTTOM: {
+				type: 'glue-bottom',
+				addEntryPoint: function () {
+					$('body').append($banner);
+					$banner.addClass('initialized');
+				}
+			}
+		},
+		track = tracker.buildTrackingFunction({
+			category: 'potential-member-experiment',
+			trackingMethod: 'analytics'
+		}),
+		viewabilityCounter = 0,
+		viewabilityInterval;
+
+	function init() {
+		if ($.cookie(dismissCookieName)) {
+			return;
+		}
+
+		if (shouldSetExperiment()) {
+			$.when(
+				loader({
+					type: loader.MULTI,
+					resources: {
+						mustache: '/extensions/wikia/PotentialMemberPageExperiments/templates/PMPEntryPoint.mustache',
+						styles: '/extensions/wikia/PotentialMemberPageExperiments/styles/entry-point-experiment.scss'
+					}
+				})
+			).done(setExperiment);
+		}
+	}
+
+	function shouldSetExperiment() {
+		return experiments.hasOwnProperty(experimentGroup);
+	}
+
+	function setExperiment(resources) {
+		var experiment = experiments[experimentGroup];
+
+		loader.processStyle(resources.styles);
+
+		$banner = $(mustache.render(resources.mustache[0], { bannerType: experiment.type }))
+			.on('click', '.pmp-entry-point-button', onEntryPointClick)
```

Comment:
Since this is a link that navigates away, the event should be `mousedown touchstart` to give the browser time to send the event naturally.